### ISSUE TITLE
Develop a plan for simplifying image management for our hubs

### ISSUE THREAD

**Issue by balajialg** [berkeley-dsep-infra/hubs#2669]

**Comment by balajialg** [comment]
@yuvipanda Are the labels for this issue accurate?

**Comment by yuvipanda** [comment]
Ah, I almost never use labels in issues as I find them mostly very ambiguous. If those are helpful to you, then I'm happy to keep them here!

**Comment by balajialg** [comment]
@yuvipanda Yes, it will be helpful for me if we keep adding labels. I am mostly using labels as a proxy for the complexity of the task involved (which I suppose is not accurate).
